### ISSUE TITLE
psm() generates invalid argument

### ISSUE THREAD

**Issue by khamyl** [thiagoalessio/tesseract-ocr-for-php#229]
### Expected behavior
```php
<?php
$tesseract = new TesseractOCR($img_fpath);
$tesseract->lang('slk');
$tesseract->psm(1); //Automatic page segmentation with OSD (orientation and script detection) [Default: 3]
$tesseract->config('tessedit_create_tsv', '1')
          ->config('tessedit_ocr_engine_mode', '1'); //Neural nets LSTM only.
$OCR_res = $tesseract->run();
```

I expect it to run the OCR. But it generates an invalid parameter. `-psm` probably intead of `--psm`.

### Actual behavior

_Error: unknown command line argument '-psm'_

### Steps to reproduce the behavior

Add the tesseract command generated by the library, example:

```php
<?php
$tesseract = new TesseractOCR($img_fpath);
$tesseract->lang('slk');
$tesseract->psm(1); //Automatic page segmentation with OSD (orientation and script detection) [Default: 3]
$tesseract->config('tessedit_create_tsv', '1')
          ->config('tessedit_ocr_engine_mode', '1'); //Neural nets LSTM only.
$OCR_res = $tesseract->run();
```
#### Environment

* Operating system: Windows 10
* PHP version: PHP 8.1.2 (cli) (built: Jan 19 2022 10:18:23) (ZTS Visual C++ 2019 x64)
* Tesseract version: tesseract v5.0.0-alpha.20200328
* tesseract-ocr-for-php version: * 2.12.0

**Comment by thiagoalessio** [comment]
thanks for the report @khamyl ! There were some changes in tesseract 5 CLI that are still not supported by this wrapper ATM. but I'll get to it some time this week and let you know once it is resolved.

**Comment by thiagoalessio** [comment]
@khamyl I'm not able to reproduce your issue. Added the following test: https://github.com/thiagoalessio/tesseract-ocr-for-php/commit/530e1204ea6c8344571950d02a350719881ece08

and it does generate the argument with 2 dashes (`--psm`): https://github.com/thiagoalessio/tesseract-ocr-for-php/runs/5988701639?check_suite_focus=true#step:5:17

Could you please share the whole command generated by your script?
```diff
  $tesseract = new TesseractOCR($img_fpath);
  $tesseract->lang('slk');
  $tesseract->psm(1); //Automatic page segmentation with OSD (orientation and script detection) [Default: 3]
  $tesseract->config('tessedit_create_tsv', '1')
            ->config('tessedit_ocr_engine_mode', '1'); //Neural nets LSTM only.
- $OCR_res = $tesseract->run();
+ echo "${tesseract->command}";
```

**Comment by khamyl** [comment]
Well, today I cannot simulate it either and it is working fine. 

But I did today probably one important bug fix on my side, where `$command->getTesseractVersion();` was returning to me _"macro"_. This was because `exec()` was running the `cmd` command and that was implicitly returning "Invalid macro definition." at the first row. I fixed that today and it is returning something that looks like real version. It may be that because of this error I was provided with CLI of some fallback version.

It is closed for me now. Thank you for your time. And sorry for bothering you.
